packages/@vue/cli-service/lib/commands/serve.js

```diff
@@ -10,12 +10,15 @@ const defaults = {
   https: false
 }
 
+const clipboardy = require('clipboardy')
```

Comment:
This should be lazy-required only when copy is needed

Reply:
Fixed.